EMML/6001-7000/EMML6451.xml

```diff
@@ -30,88 +30,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                                <title type="incomplete" ref="LIT5086TUsixLal"/>
-                            </msItem>
-                            
-                            <msItem xml:id="ms_i1.10">
-                                <locus from="49va" to="100rb"/>
-                                <title type="incomplete" ref="LIT4922HagVit"/>
-                                <incipit/>
-                            </msItem>
-                            <msItem xml:id="ms_i1.11">
+                            <msItem xml:id="ms_i1.1">
                                 <locus from="100rb" to="101va"/>
                                 <title type="complete" ref="LIT4924HagMir1"> </title>
                             </msItem>
-                            <msItem xml:id="ms_i1.12">
+                            <msItem xml:id="ms_i1.2">
                                 <locus from="101va" to="103va"/>
                                 <title type="complete" ref="LIT4925HagMir2"/>
                             </msItem>
                             
-                            <msItem xml:id="ms_i1.13">
+                            <msItem xml:id="ms_i1.3">
                                 <locus from="103va" to="104vb"/>
                                 <title type="complete" ref="LIT4926HagMir3"/>
                             </msItem>
-                            <msItem xml:id="ms_i1.14">
+                            <msItem xml:id="ms_i1.4">
                                 <locus from="104vb" to="106va"/>
                                 <title type="complete" ref="LIT4927HagMir4"/>
                             </msItem>
-                            <msItem xml:id="ms_i1.15">
+                            <msItem xml:id="ms_i1.5">
                                 <locus from="106va" to="107ra"/>
                                 <title type="complete" ref="LIT4928HagMir5"/>
                             </msItem>
-                            <msItem xml:id="ms_i1.16">
+                            <msItem xml:id="ms_i1.6">
                                 <locus from="107ra" to="107vb"/>
                                 <title type="complete" ref="LIT4929HagMir6"/>
                             </msItem>
-                            <msItem xml:id="ms_i1.17">
+                            <msItem xml:id="ms_i1.7">
                                 <locus from="107vb" to="113rb"/>  
                                 <title type="complete" ref="LIT4930HagVirt"/>                                
                             </msItem>
                                 <colophon xml:id="coloph1"><note>colophon</note></colophon>
                             </msItem>
                             <msItem xml:id="ms_i2">
                                 <locus from="113vb" to="115rb"/>
-                                <title>antiphons</title>
+                                <title>Antiphons for Jesus </title>
```

Comment:
These are antiphons from Ṣoma Dəggʷā, starting in the middle of the antiphons Maundy Thursday and ending in the (early) middle of the antiphons for Holy Saturday :)

incomplete LIT2361somade#MaundyThursday, LIT2361somade#GoodFriday, LIT2361somade#HolySaturday